roles/custom/matrix-dendrite/defaults/main.yml

```
@@ -194,3 +194,5 @@ matrix_dendrite_configuration_extension: "{{ matrix_dendrite_configuration_exten
 # Holds the final Dendrite configuration (a combination of the default and its extension).
 # You most likely don't need to touch this variable. Instead, see `matrix_dendrite_configuration_yaml`.
 matrix_dendrite_configuration: "{{ matrix_dendrite_configuration_yaml | from_yaml | combine(matrix_dendrite_configuration_extension, recursive=True) }}"
+
+auto_join_rooms: []
```

```suggestion
matrix_dendrite_userapi_auto_join_rooms: []
```

Although also relocating it elsewhere in this `main.yml` file will be better.

These `matrix_dendrite_configuration` variables should probably be last.